plugins/system_upgrade.py

```diff
@@ -60,8 +60,8 @@
 RELEASEVER_MSG = _(
     "Need a --releasever greater than the current system version.")
 DOWNLOAD_FINISHED_MSG = _(  # Translators: do not change "reboot" here
-    "Download complete! Use 'dnf system-upgrade reboot' to start the upgrade.\n"
-    "To remove cached metadata and transaction use 'dnf system-upgrade clean'")
```

Comment:
There is a similar message in check_upgrade that needs fixing: "use 'dnf system-upgrade reboot' to begin the upgrade"